templates/dag/dag.dag.condor.sub

```diff
@@ -32,7 +32,8 @@ delegate_job_GSI_credentials_lifetime = 0
 {{lines|join("\n")}}
 
 {% if generate_email_summary %}
-notify= {{email_to}}
+notify = "{{email_to}}"
```

Comment:
Looking at the docs, shouldn't all of these `notify` entries be `notify_user`?

https://htcondor.readthedocs.io/en/latest/man-pages/condor_submit.html

Reply:
Yes!  Thanks..